Tests/ASDisplayLayerTests.m

```diff
@@ -472,7 +472,7 @@ - (void)DISABLED_testTransaction
   layer1.displaysAsynchronously = YES;
 
   dispatch_semaphore_t displayAsyncLayer1Sema = dispatch_semaphore_create(0);
-  layer1Delegate.displayLayerBlock = ^(_ASDisplayLayer *asyncLayer) {
+  layer1Delegate.displayLayerBlock = ^UIImage *{
```

Comment:
Not sure about this one, tests are green but..

Reply:
Looks right to me :)